lib/internal/util.js

```diff
@@ -204,14 +204,16 @@ const kCustomPromisifyArgsSymbol = Symbol('customPromisifyArgs');
 function promisify(orig) {
   if (typeof orig !== 'function') {
     const errors = require('internal/errors');
-    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'original', 'function');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'orig', 'function');
```

Comment:
The other way around (rename the formal parameter) - https://nodejs.org/api/util.html#util_util_promisify_original